src/keys.rs

```diff
@@ -256,3 +256,63 @@ impl<C: CipherSuite> GroupVerifyingKey<C> {
         }
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use crate::testing::Secp256k1Sha256;
+    use crate::utils::Scalar;
+    use crate::{FromBytes, ToBytes};
+    use ark_ff::UniformRand;
+    use rand::{rngs::OsRng, RngCore};
+
+    #[test]
+    fn test_serialization() {
+        let mut rng = OsRng;
+
+        for _ in 0..100 {
```

Comment:
What is the purpose of doing this 100 times?

Reply:
I guess it's better to have it several times, and on random inputs, than a single time on fixed ones, no?

Reply:
Hmm, probably not! :)

Like: if there is an element of indeterminism and there could be a random key that causes the code to behave unexpectedly, then we should use a proper fuzzer and "do it properly". If there is not, well, then trying a 100 times changes nothing.

Why is 100 "enough" but 10 is not? Should it be 1000? Or 2^64 times?

Reply:
I agree it looks arbitrary the 100 times, but on the other hand it might not so simple to find the exact number. so why not 100 for now.